pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -208,6 +208,7 @@ stdenv.mkDerivation (finalAttrs: {
     ./0017-core-don-t-taint-on-unmerged-usr.patch
     ./0018-tpm2_context_init-fix-driver-name-checking.patch
     ./0019-systemctl-edit-suggest-systemdctl-edit-runtime-on-sy.patch
+  ] ++ lib.optional (!stdenv.hostPlatform.isMusl) [
```

Comment:
I wonder if `stdenv.hostPlatform.isLinux` would work better here as this patch only makes sense when glibc is in use?

Reply:
Isn't Musl also Linux?

Reply:
If anything, it would be `isLinux && isGnu`

Reply:
Done